### PR TITLE
Allow hiding cards and fields in general information

### PR DESCRIPTION
**Pull request by karelhala**
### Hide cards on general info page

Since we have a way to fetch just a few fields from API we should also allow hiding certain parts of general info detail. This PR allows to do that by simply adding `Wrapper` prop for each card and if consumer passes `false` in it the card won't be shown. There is also option to pass in custom Wrapper for each card to hide certain fields or to render something completely different.

```JSX
<GeneralInformation
  store={useStore()}
  writePermissions={writePermissions}
  ConfigurationCardWrapper={false}
  SystemCardWrapper={(props) => (
    <Suspense>
      <SystemCard {...props} hasSAP={false} />
    </Suspense>
  )}
  OperatingSystemCardWrapper={(props) => (
    <Suspense fallback="">
      <OperatingSystemCard {...props} hasKernelModules={false} />
    </Suspense>
  )}
/>
```

### CODE REVIEW

File: packages/inventory-general-info/src/GeneralInformation/GeneralInformation.js [hunk 1]
```diff
@@ -111,10 +122,22 @@ GeneralInformation.propTypes = {
...
     loadSystemDetail: PropTypes.func,
     store: PropTypes.any,
-    writePermissions: PropTypes.bool
+    writePermissions: PropTypes.bool,
+    SystemCardWrapper: PropTypes.oneOfType([ PropTypes.node, PropTypes.bool ]),
+    OperatingSystemCardWrapper: PropTypes.oneOfType([ PropTypes.node, PropTypes.bool ]),
+    BiosCardWrapper: PropTypes.oneOfType([ PropTypes.node, PropTypes.bool ]),
+    InfrastructureCardWrapper: PropTypes.oneOfType([ PropTypes.node, PropTypes.bool ]),
+    ConfigurationCardWrapper: PropTypes.oneOfType([ PropTypes.node, PropTypes.bool ]),
+    CollectionCardWrapper: PropTypes.oneOfType([ PropTypes.node, PropTypes.bool ])
```
Comment (rvsia) [inline packages/inventory-general-info/src/GeneralInformation/GeneralInformation.js:131]: Why `bool`? 🤔

Comment (karelhala) [reply packages/inventory-general-info/src/GeneralInformation/GeneralInformation.js:131]: So consumers can rutn off the wrapper.
```JSX
<GeneralInformation
  store={useStore()}
  OperatingSystemCardWrapper={false}
  BiosCardWrapper={false}
  InfrastructureCardWrapper={false}
  ConfigurationCardWrapper={false}
  CollectionCardWrapper={false}
/>
```

Will show just bios card

Comment (rvsia) [reply packages/inventory-general-info/src/GeneralInformation/GeneralInformation.js:131]: hmmm I would rather use `null` (= `Node`)

Comment (karelhala) [reply packages/inventory-general-info/src/GeneralInformation/GeneralInformation.js:131]: Yeah, that works as well.

**Review by rvsia — APPROVED** [review]

File: packages/inventory-general-info/src/GeneralInformation/GeneralInformation.js (hunk 1 above)
Comment (rvsia) [reply packages/inventory-general-info/src/GeneralInformation/GeneralInformation.js:131]: Yeah, I understand, I just don't like the false/bool here.. 😄  but it doesn't matter too much.

**Pull request merged**